### ISSUE TITLE
Crash in Release Mode

### ISSUE THREAD

**Issue by OneFingerCodingWarrior** [CommunityToolkit/Graph-Controls#145]
## Describe the bug
1、CommunityToolkit.Graph.UWP has the reference of Microsoft.Toolkit.Uwp.UI.Controls.Input (>= 7.0.2)
2、which caused exception in release mode  System.MissingMethodException: 'Method 'XamlMetaDataProvider..ctor()' from assembly 'Microsoft.Toolkit.Uwp.UI.Controls.Input' was not included in compilation, but was referenced in XamlTypeInfoProvider.OtherProviders.get(). There may have been a missing assembly, or a dependency on a more recent Windows SDK release.'

- [ ] Is this bug a regression in the toolkit?  If so, what toolkit version did you last see it work:

## Environment

```
NuGet Package(s): 

Package Version(s): 

Windows 10 Build Number:
- [ ] Fall Creators Update (16299)
- [ ] April 2018 Update (17134)
- [ ] October 2018 Update (17763)
- [ ] May 2019 Update (18362)
- [ ] Insider Build (build number: )

App min and target version:
- [ ] Fall Creators Update (16299)
- [ ] April 2018 Update (17134)
- [ ] October 2018 Update (17763)
- [ ] May 2019 Update (18362)
- [ ] Insider Build (xxxxx)

Device form factor:
- [ ] Desktop
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

Visual Studio 
- [ ] 2017 (version: )
- [ ] 2019 (version: ) 
- [ ] 2019 Preview (version: )

```

**Comment by shweaver-MSFT** [comment]
@OneFingerCodingWarrior what version of the CommunityToolkit.Graph.Uwp package are you using?

I'm able to do a release build of the SampleTest app included in the solution without error. However, I did just make some notable changes to the solutions target and props files earlier today, which might resolve your issue.

There should be new nuget packages available for you to try in the preview feed. Try out the latest and let me know if you are still hitting this issue: https://github.com/CommunityToolkit/WindowsCommunityToolkit/wiki/Preview-Packages

**Comment by OneFingerCodingWarrior** [comment]
i create a new project, and using the latest preview build
![image](https://user-images.githubusercontent.com/53161726/129812013-84836950-44b8-4427-ad94-4dc39414d078.png)
i will try today's preview build

**Comment by OneFingerCodingWarrior** [comment]
looks like there is no new preview today
![image](https://user-images.githubusercontent.com/53161726/129823175-5e058dbe-e1fa-48f1-aa8e-781364d22733.png)

**Comment by OneFingerCodingWarrior** [comment]
1、I download the latest main branch source code
2、UwpMsalProviderSample works fine in Release mode
3、I add the Microsoft.Toolkit.Uwp.UI.Contro(7.1.0-preview1) to the project. And add DropShadowPanel to the MainPage.xaml
during the compilation, i got the following warnings
1>E:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.net.native.compiler\2.2.9-rel-29512-01\tools\Microsoft.NetNative.targets(805,5): warning : ILTransform : warning ILT0005: Type 'Microsoft.Toolkit.Uwp.UI.Controls.Input.Microsoft_Toolkit_Uwp_UI_Controls_Input_XamlTypeInfo.XamlMetaDataProvider' from assembly 'Microsoft.Toolkit.Uwp.UI.Controls.Input' was not included in compilation, but was referenced in method 'XamlTypeInfoProvider.OtherProviders.get()'. There may have been a missing assembly, or a dependency on a more recent Windows SDK release.
1>E:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.net.native.compiler\2.2.9-rel-29512-01\tools\Microsoft.NetNative.targets(805,5): warning : ILTransform : warning ILT0003: Method 'XamlTypeInfoProvider.OtherProviders.get()' will always throw an exception due to the missing method 'XamlMetaDataProvider..ctor()' in assembly 'Microsoft.Toolkit.Uwp.UI.Controls.Input'. There may have been a missing assembly, or a dependency on a more recent Windows SDK release.
4、UwpMsalProviderSample crash in Release mode

**Comment by OneFingerCodingWarrior** [comment]
after updating CommunityToolkit.Graph.UWP 's dependence Microsoft.Toolkit.Uwp.UI.Controls.Input  from  7.0.2 to 7.1.0-preview1
app back to normal in Release mode

**Comment by shweaver-MSFT** [comment]
Hey @OneFingerCodingWarrior sorry for the run around, I should have been more specific. Using the MainLatest feed from WindowsCommunityToolkit, try installing the 7.1.0-build.43. That is the most recent build, which includes the build changes I mentioned. 

But you are so right, I tested with the 7.1.0-preview1 nuget package and I see the same errors as you. Sorry that you ran into this. 

It's a bit funny, the build changes I made resolved the errors even without updating the reference to Microsoft.Toolkit.Uwp.Controls.Input. I'll get that dependency version updated and push an update though.

Feed URL: https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-MainLatest/nuget/v3/index.json

![image](https://user-images.githubusercontent.com/25832310/129965914-a5b3771b-a5e9-4ece-b6e5-16d24b776a1b.png)

![image](https://user-images.githubusercontent.com/25832310/129965945-c3ceee95-51e8-4213-a063-d720c07bac53.png)
